admin_manual/groupware/contacts.rst

```diff
@@ -42,3 +42,10 @@ Address Book Sync
 ^^^^^^^^^^^^^^^^^
 
 The address book is updated automatically with every added, modified, disabled or removed user. Admins can also trigger a full rewrite of the address book :ref:`with occ<occ-dav-sync-system-address-book>`.
+
+Shared items
+------------
+
+.. versionadded:: 5.5.0
+
+.. note:: `The related resources app <https://apps.nextcloud.com/apps/related_resources>`_ should be already in place to enable this feature. By default the app is shipped with Nextcloud.
```

Comment:
Let's rephrase saying that the app is shipped but has to be *enabled* for the feature to be available

This new section has only a note element, no actual contents. Let's render the text without a note box.

Reply:
:heavy_check_mark: